spec/controllers/idv/scan_id_acuant_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+describe Idv::ScanIdAcuantController do
+  before do |test|
+    stub_sign_in unless test.metadata[:skip_sign_in]
+    allow(Figaro.env).to receive(:enable_mobile_capture).and_return('true')
+  end
+
+  describe '#field_image' do
+    it 'works' do
+      get :field_image, params: { instance_id: 'foo' }
```

Comment:
hi, this is another "it works" test that isn't clear what it's testing, can we specify something about the output for this test? and the others in this file too

Reply:
Ok.  Added better descriptions.  Working on output...